demo/client/backend/mock/chaincode/fpc_chaincode.go

```diff
@@ -10,7 +10,7 @@ package chaincode
 
 import (
 	"github.com/hyperledger-labs/fabric-private-chaincode/ecc"
-	"github.com/hyperledger/fabric/core/chaincode/shim"
+        "github.com/hyperledger/fabric-chaincode-go/shim"
```

Comment:
is this correctly indented?

Reply:
this fixes a left over from #247

Reply:
got tabs and spaces mixed here. probably needs a gofmt.

Reply:
Indeed. Given that demo is now disabled in the build, make checks didn't catch it.  Just pushed an update